packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -752,23 +771,7 @@ ensureIndex(Posts,
   }
 );
 
-Posts.addView("afRecentDiscussionThreadsList", terms => {
-  return {
-    selector: {
-      baseScore: {$gt:0},
-      hideFrontpageComments: false,
-      af: true,
```

Comment:
I don't see a corresponding af tag in the new afRecentDiscussion – is that just unnecessary because Alignment Forum already filters that at a higher level?

Reply:
Yes